src/Tribe/Plugins_API.php

```diff
@@ -136,27 +162,28 @@ public function filter_api_result( $result, $action, $args ) {
 	 *
 	 * @since 4.5.3
 	 *
-	 * @param array $product_data
+	 * @param array  $product_data
+	 * @param string $product_slug
 	 *
 	 * @return array
 	 */
-	public function build_product_data( $product_data ) {
+	public function build_product_data( $product_data, $product_slug ) {
 		$defaults = array(
 			'name'                     => null,
 			'slug'                     => null,
 			'version'                  => null,
 			'author'                   => '<a href="http://m.tri.be/19o3">Modern Tribe, Inc.</a>',
 			'author_profile'           => null,
 			'requires'                 => '3.9',
-			'tested'                   => '4.7.5',
-			'rating'                   => null,
+			'tested'                   => $GLOBALS['wp_version'],
```

Comment:
Actually that's not a bad thought ... that said, if we can confirm that Zach's suggestion is valid this may mostly be moot (since we'll then generally be able to override this default with the correct and current information)